### DISCUSSION TITLE
.cshtml, .razor files

### DISCUSSION THREAD

**Discussion by na1307** [community/community#122860] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I would like to see Razor added to code search. The [Razor syntax](https://learn.microsoft.com/en-us/aspnet/core/mvc/views/razor?view=aspnetcore-8.0) is used in .cshtml and .razor files and coexists with C#.

**Comment by 22coffeeboi** [comment]
Razor syntax, commonly used in .cshtml and .razor files alongside C#, allows for dynamic content rendering in web applications. Adding support for Razor syntax in code search would likely enhance the search capabilities for developers working with ASP.NET or Blazor applications. This could improve efficiency and streamline development processes

**Comment by look** [comment, marked as answer]
Hi @na1307, thanks for the suggestion. In order to add this, we need a viable tree-sitter parser for Razor (perhaps the tree-sitter C# parser supports it, but I couldn't tell at a glance) and a set of `tags` queries.
